warehouse/encoding/parquetloader.go

```diff
@@ -63,44 +64,39 @@ func (loader *parquetLoader) Write() error {
 
 func parquetValue(val interface{}, colType string) (interface{}, error) {
 	switch colType {
-	case "bigint", "int":
-		retVal, err := getInt64(val)
-		return retVal, err
-	case "boolean":
-		retVal, err := getBool(val)
-		return retVal, err
-	case "float":
-		retVal, err := getFloat64(val)
-		return retVal, err
-	case "datetime":
-		retVal, err := getUnixTimestamp(val)
-		return retVal, err
-	case "string", "text":
-		retVal, err := getString(val)
-		return retVal, err
+	case model.BigIntDataType, model.IntDataType:
+		return getInt64(val)
+	case model.BooleanDataType:
+		return getBool(val)
+	case model.FloatDataType:
+		return getFloat64(val)
+	case model.DateTimeDataType:
+		return getUnixTimestamp(val)
+	case model.StringDataType, model.TextDataType:
+		return getString(val)
 	}
 	return nil, fmt.Errorf("unsupported type for parquet: %s", colType)
 }
 
 func getInt64(val interface{}) (int64, error) {
 	if intVal, ok := val.(int); !ok {
-		return 0, fmt.Errorf("failed to convert %v to int64", val)
+		return 0, errors.New("failed to convert to int64")
```

Comment:
Do you think removing these values from the error message will make debugging more difficult for us/customer. Or they were never really used, so it won't make any difference?

Reply:
I think they were never really used. Let me check from logs that do we ever got these.

Reply:
Tried to check for it in the [last 60 days](https://rudderstack.grafana.net/explore?schemaVersion=1&panes=%7B%22p3b%22:%7B%22datasource%22:%22bcdb67e5-a0f0-4bb3-b7da-840cfd0b6f43%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bcontainer%3D~%5C%22rs-warehouse-slave%7Crudderstack-warehouse-slave%5C%22%7D%20%7C%3D%20%60failed%20to%20convert%60%22,%22queryType%22:%22range%22,%22datasource%22:%7B%22type%22:%22loki%22,%22uid%22:%22bcdb67e5-a0f0-4bb3-b7da-840cfd0b6f43%22%7D,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-60d%22,%22to%22:%22now%22%7D,%22panelsState%22:%7B%22logs%22:%7B%22visualisationType%22:%22logs%22%7D%7D%7D%7D&orgId=1) but didn't find any.